sensor_msgs/Image color_image
sensor_msgs/Image depth_image
sensor_msgs/Image segmask
sensor_msgs/CameraInfo camera_info
---
geometry_msgs/Pose pose
